web/src/UserEditPage.js

```diff
@@ -115,7 +115,17 @@ class UserEditPage extends React.Component {
     return value;
   }
 
+  getEmptyText(text) {
```

Comment:
provide a screenshot of the UI effect

Reply:
![image](https://user-images.githubusercontent.com/32982627/211824783-13b3d1b0-be71-4ccd-9f02-cbb6eb8fc22f.png)
![image](https://user-images.githubusercontent.com/32982627/211824517-56dc857e-35da-480d-a56c-6d5799dac917.png)

Reply:
Don't do this. This code is fragile, and maybe don't work for other languages.